LayoutTests/TestExpectations

```diff
@@ -7501,3 +7501,71 @@ webkit.org/b/272417 imported/w3c/web-platform-tests/svg/path/property/priority.s
 
 # re-import css/css-align WPT failure
 webkit.org/b/271692 imported/w3c/web-platform-tests/css/css-align/blocks/align-content-block-break-overflow-020.html [ ImageOnlyFailure ]
+
+# Anchor positioning not yet implemented
+webkit.org/b/274926 imported/w3c/web-platform-tests/css/css-anchor-position/anchor-center-002.html [ Skip ]
```

Comment:
We can skip the whole folder with 
`webkit.org/b/274926 imported/w3c/web-platform-tests/css/css-anchor-position/ [ Skip ]`